tests/mapped_struct.py

```diff
@@ -986,6 +986,9 @@ def _testBuild(self, N, tempdir, **gen_kwargs):
                 self.assertIsNotNone(elem)
             if v not in elem:
                 self.assertIn(v, elem)
+            int_elem = rvget(k)
+            if int_elem is None:
+                self.assertIsNotNone(int_elem)
```

Comment:
This looks weirdly contradictory.

I'm sure you wanted something else.

Also, it shouldn't be on the base class. You have to do this in `ApproxIntegerIdMultiMapperTest` only.

Reply:
I was just following the previous tests, IDK why it was done that way (check the previous lines, it's the same and the assert above that too)